features/collections/collection_participants.feature

```diff
@@ -66,3 +66,14 @@
     And I follow "Join"
   Then I should see "You are now a member of Such a nice collection"
   When I am in the default browser
+
+Scenario: Collection member should see correct button text
+Given I have joined the collection "ModeratedCollection" as "sam"
```

Comment:
I looked at the step definition for this, and it looks like the step doesn't create the collection -- just looks for and joins an existing one. So you'd need to start with `Given I have the collection "ModeratedCollection"` to create the collection.